starters/solidjs-tailwind/src/components/CounterExample/Counter.spec.jsx

```diff
@@ -1,4 +1,4 @@
-import { fireEvent, render, screen } from 'solid-testing-library';
+import { fireEvent, render, screen } from '@solidjs/testing-library';
```

Comment:
Testing-library [recommends](https://testing-library.com/docs/user-event/intro) using the [user-event](https://github.com/testing-library/user-event) module for most user-interactions and falling back to fireEvent for things user-event can't do - we should do a follow up PR for this